tests/e2e/e2e_test.go

```diff
@@ -11,45 +11,47 @@ import (
 	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
 )
 
-// func (s *IntegrationTestSuite) TestQueryBalances() {
-// 	var (
-// 		expectedDenoms    = []string{osmoDenom, stakeDenom}
-// 		expectedBalancesA = []uint64{osmoBalanceA, stakeBalanceA - stakeAmountA}
-// 		expectedBalancesB = []uint64{osmoBalanceB, stakeBalanceB - stakeAmountB}
-// 	)
-
-// 	chainAAPIEndpoint := fmt.Sprintf("http://%s", s.valResources[s.chainA.id][0].GetHostPort("1317/tcp"))
-// 	balancesA, err := queryBalances(chainAAPIEndpoint, s.chainA.validators[0].keyInfo.GetAddress().String())
-// 	s.Require().NoError(err)
-// 	s.Require().NotNil(balancesA)
-// 	// s.Require().Equal(2, len(balancesA))
-
-// 	chainBAPIEndpoint := fmt.Sprintf("http://%s", s.valResources[s.chainB.id][0].GetHostPort("1317/tcp"))
-// 	balancesB, err := queryBalances(chainBAPIEndpoint, s.chainB.validators[0].keyInfo.GetAddress().String())
-// 	s.Require().NoError(err)
-// 	s.Require().NotNil(balancesB)
-// 	// s.Require().Equal(2, len(balancesB))
-
-// 	actualDenomsA := make([]string, 0, 2)
-// 	actualBalancesA := make([]uint64, 0, 2)
-// 	actualDenomsB := make([]string, 0, 2)
-// 	actualBalancesB := make([]uint64, 0, 2)
-
-// 	for _, balanceA := range balancesA {
-// 		actualDenomsA = append(actualDenomsA, balanceA.GetDenom())
-// 		actualBalancesA = append(actualBalancesA, balanceA.Amount.Uint64())
-// 	}
-
-// 	for _, balanceB := range balancesB {
-// 		actualDenomsB = append(actualDenomsB, balanceB.GetDenom())
-// 		actualBalancesB = append(actualBalancesB, balanceB.Amount.Uint64())
-// 	}
-
-// 	s.Require().ElementsMatch(expectedDenoms, actualDenomsA)
-// 	s.Require().ElementsMatch(expectedBalancesA, actualBalancesA)
-// 	s.Require().ElementsMatch(expectedDenoms, actualDenomsB)
-// 	s.Require().ElementsMatch(expectedBalancesB, actualBalancesB)
-// }
+func (s *IntegrationTestSuite) TestQueryBalances() {
+	var (
+		expectedDenomsA   = []string{osmoDenom, stakeDenom}
+		expectedDenomsB   = []string{osmoDenom, stakeDenom, ibcDenom}
+		expectedBalancesA = []uint64{osmoBalanceA - ibcSendAmount, stakeBalanceA - stakeAmountA}
+		expectedBalancesB = []uint64{osmoBalanceB, stakeBalanceB - stakeAmountB, ibcSendAmount}
+	)
+
+	chainAAPIEndpoint := fmt.Sprintf("http://%s", s.valResources[s.chainA.id][0].GetHostPort("1317/tcp"))
+	balancesA, err := queryBalances(chainAAPIEndpoint, s.chainA.validators[0].keyInfo.GetAddress().String())
+	s.Require().NoError(err)
+	s.Require().NotNil(balancesA)
+	// s.Require().Equal(2, len(balancesA))
```

Comment:
and this one

Reply:
So why is this commented out? Is this because or test execution order?